x-pack/plugins/infra/public/pages/metrics/inventory_view/hooks/use_metrics_hosts_anomalies.ts

```diff
@@ -309,8 +309,10 @@ export const useMetricsHostsAnomaliesResults = ({
     }
   }, [dispatch, reducerState]);
 
-  const isLoadingMetricsHostsAnomalies = useMemo(
-    () => getMetricsHostsAnomaliesRequest.state === 'pending',
+  const isPendingMetricsHostsAnomalies = useMemo(
```

Comment:
I'd probably remove the `useMemo` here. This is not a complex logic to check, and we're trading performance for memory use.